src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -3053,14 +3053,22 @@ void processContextElement
       if (forwardsPending(responseP) == false)
       {
         cerP->statusCode.fill(SccContextElementNotFound);
-        responseP->oe.fill(SccContextElementNotFound, "No context element found", "NotFound");
+
+        if (apiVersion == "v1")
+        {
+          responseP->oe.fill(SccContextElementNotFound, "Entity not found. Check type and id", "NotFound");
```

Comment:
ok, but with it the calls looks similar between v1 and v2.

However, I just noticed that I have changed the text also for v1 here ... That's not what I intented to do ...

Rollback of that part?